src/test/resources/com/znsio/sample/e2e/features/indigoFlightPromoCode.feature

```diff
@@ -0,0 +1,9 @@
+
+  @indigo
+Feature: Indigo gift voucher with invalid promo code
+
+  @web @android
+Scenario: As a guest user, Personalise and preview an Indigo gift voucher with invalid Promo code
+  Given I as a guest user, preview and personalise "2" gift vouchers of "5000" rupees
+  When I apply a "invalid" promo code
+  Then I should able to purchase the ticket of "10000" rupees
```

Comment:
You are not purchasing a ticket. You are purchasing a gift voucher

Also, mentioning 10000 here means if you change the data in the Given step, you need to also update the data in the Then step.
I feel you can avoid mentioning 10000 in the Then step. Instead you could say "Then I should be able to purchase the gift voucher at the original price"

Reply:
Done